.github/actions/compute-sha256/README.md

```diff
@@ -0,0 +1,13 @@
+# Compute SHA256
+
+## How to build this action in development
+- Install node
+- Install typescript
+- `npm ci`
+- `npm run all`
+
+## Compare the dist/index.js with the actual source
+This is to ensure that the source is correct.
+- `git diff --ignore-space-at-eol dist diff`. In this case the prdist/index.js is the Pull Request version.
```

Comment:
is this a typo? Should it be `git diff --ignore-space-at-eol dist`?

`prdist/index.js` does not appear in the command. What does `In this case the prdist/index.js is the Pull Request version.` mean?

Reply:
Can this wait for a follow-up PR? These aren't blockers.

Reply:
It's pretty easy to do, and I won't have to review another PR. That's why I'm asking. Hope this is OK.

Reply:
This is constantly going back and forth. That is my concern. Every time something new is identified.

Reply:
I know, it's hard to catch everything at once, especially when there were so many new things in the PR (TS, etc).

Reply:
OK, I will address this one 👍

Reply:
I am going to remove this as it is addressed by presubmit action. This is redundant and confusing.